ui-components/src/blocks/InfoCard.scss

```diff
@@ -1,51 +1,47 @@
 .info-card {
-  @apply p-6;
-  @apply rounded;
-  @apply border;
-  @apply border-gray-450;
-  @apply mb-4;
-  @apply bg-white;
+  --padding: var(--bloom-s6);
+  --margin-bottom: var(--bloom-s4);
+  --border-color: var(--bloom-color-gray-450);
+  --background-color: var(--bloom-color-white);
+  --background-color-lighter: var(--bloom-color-primary-lighter);
+  --title-color: var(--bloom-color-gray-800);
+  --title-font-size: var(--bloom-font-size-base);
+  --title-font-family: var(--bloom-font-sans);
+  --title-text-transform: uppercase;
+  --title-text-transform-lighter: none;
+  --title-letter-spacing: 0.025em;
+  --title-letter-spacing-lighter: 0;
+
+  padding: var(--padding);
+  border-color: var(--border-color);
+  margin-bottom: var(--margin-bottom);
+  border-radius: 0.25rem;
+  border-width: 1px;
+  background-color: var(--background-color);
 }
 
 .info-card.is-normal-primary-lighter,
 .markdown .info-card {
-  @apply bg-primary-lighter;
+  background-color: var(--background-color-lighter);
 
   .info-card__title {
-    @apply text-base;
-    @apply normal-case;
-    @apply tracking-normal;
+    font-size: var(--title-font-size);
+    text-transform: var(--title-text-transform-lighter);
+    letter-spacing: var(--title-letter-spacing-lighter);
   }
 }
 
 .info-card__title {
-  @apply text-gray-800;
-  @apply uppercase;
-  @apply font-sans;
-  @apply font-bold;
-  @apply text-sm;
-  @apply tracking-wide;
+  color: var(--title-color);
+  text-transform: var(--title-text-transform);
+  font-weight: bold;
+  font-size: var(--title-font-size);
+  font-family: var(--bloom-font-sans);
```

Comment:
should this be `var(--title-font-family)`?

Reply:
yup! fixed

Reply:
yup! fixed. Thanks.